libs/core/src/components/pds-table/pds-table-head/pds-table-head.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, Element, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'pds-table-head',
+  styleUrl: 'pds-table-head.scss',
+  shadow: true,
+})
+export class PdsTableHead {
+  @Element() hostElement: HTMLPdsTableHeadElement;
+  tableRef: HTMLPdsTableElement
+
+  componentWillRender() {
+    this.tableRef = this.hostElement.closest('pds-table') as HTMLPdsTableElement;
+
+    if (this.tableRef && this.tableRef.fixedColumn) {
+      const tableCell = this.hostElement.querySelector('pds-table-head-cell:first-child');
+      tableCell?.classList.add("is-fixed");
+    }
+  }
+  render() {
+    return (
+      <Host role="row">
+        {this.tableRef && this.tableRef.selectable && (
+          <pds-table-checkbox-cell part={this.tableRef.fixedColumn ? 'checkbox-cell' : ''}>
+          </pds-table-checkbox-cell>
```

Comment:
This is where i think we can change this to become something like the example below.

```
<pds-table-cell part={this.tableRef.fixedColumn ? 'checkbox-cell': '')>
  <pds-checkbox labelHidden />
</pds-table-cell>
```

Reply:
Was able to remove the need for the checkbox cell here. Marking as resolved.